scripts/docs_config/introduction.rst

```diff
@@ -95,6 +95,45 @@ the `allocation API`_ as a first argument. There is also a possibility to
 retrieve a memory pool from an existing memory pointer that points to a memory 
 previously allocated by UMF.
 
+Logging
+---------------------
+
+Logging in UMF is handled by logger. There are several levels of logging: *debug*, *info*, *warning*, and *error*.
+The level of logging determines what messages will be printed, ie. the level set to *warning* means all messages at levels *warning* and *error* will be printed.
+
+By default, there is a guarantee that *error* messages are flushed immediately. One can change this behavior to flush on lower-level messages.
+
+Loggers redirect messages to *stdout*, *stderr*, or a file
+
+By default, no messages are printed. To enable logger you have to set **UMF_LOG** environment variable which have following syntax for setting logger options:
+
+  "[level:debug|info|warning|error];[flush:<debug|info|warning|error>];[output:stdout|stderr|file=<path>];[timestamp:yes|no][pid:yes|no]"
```

Comment:
There is a missing `;` between timestamp and pid

Reply:
Done